pkg/platform/abstract/platform_test.go

```diff
@@ -946,6 +946,100 @@ func (suite *AbstractPlatformTestSuite) TestCreateFunctionEvent() {
 	suite.Require().NoError(err)
 }
 
+func (suite *AbstractPlatformTestSuite) TestValidateNodeSelector() {
+	for idx, testCase := range []struct {
+		nodeSelector         map[string]string
```

Comment:
add `testName` and explain in pascal case the test case
e.g.:
`Sanity`
`FailMissingKey`
`SegmentInvalidPrefix`

and so on